applications/DEMApplication/CMakeLists.txt

```diff
@@ -98,6 +98,12 @@ set(KRATOS_DEM_APPLICATION_CORE
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/stationarity_checker.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_processes/automatic_dt_process.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/multiaxial_control_module_generalized_2d_utilities.cpp
+
+    # processes
```

Comment:
no need of this comment. We have been mixing all types. If yout put the comment, organize the rest too, but in my opinion it is not necessary